app/main.py

```diff
@@ -1 +1,11 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
```

Comment:
Check the length of the splitted command to make sure it consists of three parts